src/main/java/pl/rpg/menu/Main.java

```diff
@@ -0,0 +1,36 @@
+package pl.rpg.menu;
+
+import pl.rpg.game.Game;
+
+import java.util.Scanner;
+
+public class Main {
+
+  public static void main(String[] args) {
+
+    Scanner scanner = new Scanner(System.in);
+    int choice;
+    String color;
```

Comment:
If someone would just press new game, this variable will be null. Passing null is very bad because it can easily crash our application (Game class will throw NullPointerException)